### PR TITLE
WIP: narrows to signing micro-service; removes status handling

### PR DESCRIPTION
**Pull request by jchartrand**
Basically makes this a micro-service that does nothing but sign, whereas it had previously signed and also invoked the status manager to allocate a status position. That previous functionality is now provided by the issuer-coordinator which invokes this service and the status-service to handle the full issuance.

Putting this up here as a draft PR just in case anyone wants to take a look at it before the final PR goes in.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -45,38 +32,11 @@ export async function build(opts = {}) {
...
-                console.log(error);
-                return res.status(500).json(error);
-            }
-        })
-
-        app.get('/seedgen', async (req, res, next) => {
-            const newSeed = await generateSeed()
-            res.send(newSeed)
-        });
+    app.get('/seedgen', async (req, res, next) => {
```
Comment (dmitrizagidulin) [inline src/app.js:35]: what's this one for?

Comment (jchartrand) [reply src/app.js:35]: This is a convenience endpoint that will generate a new seed, as explained here: https://github.com/digitalcredentials/signing-service/blob/8ae3e5e0b2a919c90bade0eb6d9c5c91df2c5f63/README.md#signing-key

It just generates a new random seed and returns it, along with the associated DID:KEY and DID document.

File: README.md [hunk 2]
```diff
@@ -1,56 +1,180 @@
...
+There is a sample .env file provided called .env.example to help you get started with your own .env file. The supported fields:
 
-1.  Install Docker.
+| Key | Description | Default | Required |
+| --- | --- | --- | --- |
+| `PORT` | http port on which to run the express app | 4007 | no |
+| `ENABLE_HTTPS_FOR_DEV` | runs the dev server over https - ONLY FOR DEV - typically to allow CORS calls from a browser | false | no |
+| `TENANT_SEED_{TENANT_NAME}` | see [tenants](#tenants) section for instructions | no | no |
+
+### Tenants
```
[the remark this answers is not in the document]
Comment (jchartrand) [reply README.md:64]: I started out with 'instance', but every time I went to explain it in the README, or to comment on  an environment variable, I always seemed to keep coming back to having to say things like, 'an instance is like a tenant'. It feels like more people are familiar with what a 'tenant' has come to mean. Happy to switch back to 'instance' if needed.

**Comment by jchartrand** [comment]
Merging this in for now so that the repo matches what's in DockerHub. Can revert if needed, or update, in particular to adjust the tenant/instance naming.

**Pull request merged**